src/graphql/checkoutLineItemsReplaceMutation.graphql

```diff
@@ -1,7 +1,7 @@
 mutation ($checkoutId: ID!, $lineItems: [CheckoutLineItemInput!]!) {
   checkoutLineItemsReplace(checkoutId: $checkoutId, lineItems: $lineItems) {
     userErrors {
-      ...UserErrorFragment
+      ...CheckoutUserErrorFragment
```

Comment:
Does this not have the checkoutUserErrors field because it's newer than the other line item mutations?

Reply:
Yeah, right on. This mutation is newer so it doesn't have the legacy `userErrors`, so we don't need to distinguish between the legacy `userErrors` (of type [UserError!]!) and the newer checkout user errors type ([CheckoutUserError!]!)

Reply:
https://help.shopify.com/en/api/custom-storefronts/storefront-api/reference/mutation/checkoutlineitemsreplace

Reply:
Should we leave a note about this so that someone doesn't think it's a typo and change it?